PHP/PHP Source.sublime-syntax

```diff
@@ -547,10 +608,11 @@ contexts:
           pop: true
 
   class-body:
-    - meta_content_scope: meta.class.php meta.block.php
+    - meta_scope: meta.class.php meta.block.php
+    - include: attributes
     - include: comments
     - match: '\}'
-      scope: meta.class.php meta.block.php punctuation.section.block.php
+      scope: punctuation.section.block.php
```

Comment:
I noticed this is missing the `.end` label. I searched the existing state of the syntax and there are a handful of other situations where `.begin` and `.end` are missing. Perhaps change them all?

```suggestion
      scope: punctuation.section.block.end.php
```